### ISSUE TITLE
Git log returning string "%aI" instead of a date

### ISSUE THREAD

**Issue by EskelCz** [steveukx/git-js#871]
I'm simply running `repository.log({ n: 20 })`, and this is what is returned:
<img width="1194" alt="image" src="https://user-images.githubusercontent.com/1198558/206239783-2c48b311-ab2c-4f16-ad9e-ef79ac76fb50.png">

In console and gitlab the dates are there.
<img width="643" alt="image" src="https://user-images.githubusercontent.com/1198558/206240770-8d1b0588-b471-485e-bfe2-fa15a77814b9.png">

Using version 3.7.1.

Is this some kind of misconfiguration, or a bug of simple-git?
Thanks for any suggestions.

**Comment by steveukx** [comment]
Which version of git are you using? Try running:

```
git --version
```

The `%aI` token returns the [author date as a strict ISO 8861 format](https://git-scm.com/docs/git-log#Documentation/git-log.txt-emaIem) and `%D` token returns [ref names without the " (", ")" wrapping](https://git-scm.com/docs/git-log#Documentation/git-log.txt-emDem).

Both tokens have been available since at least git v2.19.0 and should work as expected. If you are using an older version of git, you could try switching out the format tokens:

```typescript
repository.log({
  n: 20,
  format: {
    date: '%ai',
    refs: '%d',
    hash: '%h',
  }
})
```

With format token placeholders appropriate for the version of git you're running (see git official docs at https://git-scm.com/docs/pretty-formats#Documentation/pretty-formats.txt )

**Comment by EskelCz** [comment]
@steveukx Seems like the server is using rather old version `1.8.3.1`. It's not even listed in the documentation :)
That's it then, I'll ask for an update. Thanks a lot!
